test/test_browser.py

```diff
@@ -5065,22 +5084,25 @@ def test_single_file_html(self):
     self.assertNotExists('test.mem')
 
   # Tests that SINGLE_FILE works as intended in generated HTML with MINIMAL_RUNTIME
+  @parameterized({
+    '': ([],),
+    'O3': (['-sWASM=0'],)
+  })
   @parameterized({
     '': ([],),
     'wasm2js': (['-sWASM=0'],)
   })
-  def test_minimal_runtime_single_file_html(self, args):
-    if args:
+  def test_minimal_runtime_single_file_html(self, args, opts):
```

Comment:
stacking these is slightly confusing.... does the duplication of the empty-string version mean the test will get run the same way twice?

Reply:
It seems to behave in the way you would imagine:

```
$ ./test/runner browser.test_minimal_runtime_single_file_html* --browser=0
Test suites:
['test_browser']
Running test_browser: (4 tests)
(checking sanity from test runner)
shared:INFO: (Emscripten: Running sanity checks)

Running the browser tests. Make sure the browser allows popups from localhost.

test_minimal_runtime_single_file_html (test_browser.browser.test_minimal_runtime_single_file_html) ... ok
test_minimal_runtime_single_file_html_O3 (test_browser.browser.test_minimal_runtime_single_file_html_O3) ... ok
test_minimal_runtime_single_file_html_wasm2js (test_browser.browser.test_minimal_runtime_single_file_html_wasm2js) ... ok
test_minimal_runtime_single_file_html_wasm2js_O3 (test_browser.browser.test_minimal_runtime_single_file_html_wasm2js_O3) ... ok

----------------------------------------------------------------------
Ran 4 tests in 1.199s

```